docs/.vuepress/sidebar.js

```diff
@@ -93,7 +93,8 @@ module.exports = {
           title: 'Other customizations 🆕',
           children: [
             [ 'guide/custom-schema-field-types.md', 'Custom field types' ],
-            'guide/custom-login-requirements.md'
+            'guide/custom-login-requirements.md',
+            'guide/editing-custom-widgets-in-context.md'
```

Comment:
Although technical and "Advanced", putting this information here feels disconnected from the overall Widgets and Areas part of the Guides. I feel like it should be moved under that umbrella, but that might just be My Opinion

Reply:
I don't disagree actually, I think the whole idea of Advanced doesn't help people find what they came for, but right now this clearly qualifies according to the way we've broken it down so far, so I'd rather do a separate PR to undo that for all.